app/_includes/md/konnect/okta-sso.md

```diff
@@ -0,0 +1,257 @@
+<!-- used in the Dev Portal Okta SSO how to and the Org Okta SSO how to -->
+## Prerequisites
+{% if include.desc == "Dev Portal" %}
+* Ensure that any users that need to use the Dev Portal SSO are added to Okta
+{% endif %}
+{% if include.desc == "Konnect Org" %}
+* Ensure that any users that need to use {{site.konnect_short_name}} SSO are added to Okta
+{% endif %}
+* To set up Okta single sign-on (SSO) for {{site.konnect_short_name}}, you need access to an Okta admin account and a {{site.konnect_short_name}} admin account, which you will access concurrently.
+* Optionally, if you want to use team mappings, you must configure Okta to include group attributes.
+
+## Configure an application and group claims in Okta
+
+{% navtabs %}
+{% navtab OIDC %}
+1. Create a [new OIDC application in Okta](https://help.okta.com/oie/en-us/content/topics/apps/apps_app_integration_wizard_oidc.htm) to manage {{site.konnect_saas}} account integration. Configure the following settings:
+    * **Application Type**: Web Application
+    * **Grant type**: Authorization Code
+    {% if include.desc == "Konnect Org" %}
+    * **Sign-in redirect URIs**: `https://cloud.konghq.com/login` (This is a placeholder value that you'll replace later)
+    * **Sign-out redirect URIs**: `https://cloud.konghq.com/login` (This is a placeholder value that you'll replace later)
+    {% endif %}
+    {% if include.desc == "Dev Portal" %}
```

Comment:
For reviewers: This if include.desc isn't working in the output. It's showing the Konnect Org content on both the Dev Portal pages and Konnect Org pages. Any idea what I'm doing wrong? I didn't get any errors in the Terminal when I ran this locally.